xml/decoders/piko/cv897-1024_v5.1.xml

```diff
@@ -1456,4 +1456,3 @@
 		<tooltip xml:lang="de">Effekt Flag A3 "Ausblenden"</tooltip>
 	</variable>
 </variables>
-

```

Comment:
This is the opposite to what you need to do. You have removed the last empty line, but instead you need to ensure that the empty line is there.

Reply:
Danke für den Hinweis, ich wusste nicht, das die letzte Zeile eine Leerzeile sein muss. Ich nahm an, das die letzte Zeile nur mit einem new line enden muss.
Bei mit hat es trotzdem funktioniert.

Soll ich jetzt nochmal alle xml dateien prüfen und neu hochladen?

Gruß Uli

Reply:
Thanks for the note. I didn't know that the last line has to be a empty line. I assumed that the last line must only end with a new line.
It still worked for me.

Should I now check all xml files again and upload them again?

Greetings Uli

Reply:
Check these files and upload them:

./xml/decoders/piko/Dateiliste_piko_XP5.1.txt
./xml/decoders/piko/Dateiliste_piko_4.1.txt
./xml/decoders/Piko_SmartDecoder_XP5.1_Sound.xml
./xml/decoders/piko/cv897-1024_v5.1.xml
./xml/decoders/Piko_SmartDecoder_4.1_Sound.xml